CONTRIBUTING.md

```diff
@@ -1,39 +1,33 @@
 Contributing
 ============
 
-Thank you for considering contributing to Mastodon 🐘
+Thank you for considering contributing to Florence Mastodon
```

Comment:
Might be better phrased "Thank you for your interest in contributing to Florence Mastodon"